x-pack/plugins/apm/server/routes/services/get_service_transaction_groups.ts

```diff
@@ -5,30 +5,32 @@
  * 2.0.
  */
 
-import { rangeQuery, kqlQuery } from '@kbn/observability-plugin/server';
+import { kqlQuery, rangeQuery } from '@kbn/observability-plugin/server';
```

Comment:
We now have both services and transactions, I think it's a bit confusing the name of this file, what does it return service or transaction?! Maybe we should rename it?

Reply:
it returns the transaction groups for a service. not sure if I understand what you mean? What is unclear about it?

Reply:
IMO this should call, `getTransactionGroups`, **service** is just another filter along others like **environment**, **timerange** etc...

Reply:
Hmm I think this is fine, it does what it says on the can: it gets the transaction groups for a service. Making this generic would mean that we would make `serviceName` optional and add more optional parameters I guess? Which IMHO isn't an improvement.